graql/language/compute.feature

```diff
@@ -1,17 +0,0 @@
-#
-# Copyright (C) 2020 Grakn Labs
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
-Feature: Graql Compute Query
```

Comment:
Let's not delete this file. It's a placeholder for us to do compute queries in the (near) future.

Reply:
Restored file and added placeholder note:
`This file is retained as a placeholder for future functionality`